cmd/sandbox-ce/cmd/config.go

```diff
@@ -0,0 +1,216 @@
+package cmd
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"path/filepath"
+	"strconv"
+	"time"
+
+	"github.com/pkg/errors"
+	"gopkg.in/src-d/go-cli.v0"
+)
+
+const orgName = "smacker"
+const repoName = "superset-compose"
+const defaultConfigName = "__default__"
+const composerConfigFileURL = "https://raw.githubusercontent.com/%s/%s/%s/docker-compose.yml"
+
+type configCmd struct {
+	cli.PlainCommand `name:"config" short-description:"Config"`
+}
+
+type configDownloadCmd struct {
+	Command `name:"download" short-description:"Download"`
+
+	Revision string `short:"r" long:"revision" description:"revision to download" default:"master"`
+	URL      string `short:"u" long:"url" description:"url of the config to download"`
+}
+
+func (c *configDownloadCmd) Execute(args []string) error {
+	var err error
+	var confURL, outPath string
+
+	if c.URL != "" {
+		confURL = c.URL
+		outPath, err = getConfigPath(strconv.Itoa(int(time.Now().Unix())))
+	} else {
+		confURL = getComposerConfFileURL(c.Revision)
+		outPath, err = getConfigPath(c.Revision)
+	}
+
+	if err != nil {
+		return err
+	}
+
+	err = c.downloadConfig(confURL, outPath)
+	if err != nil {
+		return err
+	}
+
+	return setDefaultConfig(outPath)
+}
+
+func (c *configDownloadCmd) downloadConfig(url, path string) error {
+	out, err := os.Create(path)
+	if err != nil {
+		return err
+	}
+
+	defer out.Close()
+
+	resp, err := http.Get(url)
+	if err != nil {
+		return err
+	}
+
+	defer resp.Body.Close()
+
+	_, err = io.Copy(out, resp.Body)
+	return err
+}
+
+type configListCmd struct {
+	Command `name:"list" short-description:"List"`
+}
+
+func (c *configListCmd) Execute(args []string) error {
+	confDirPath, err := getConfigDirPath()
+	if err != nil {
+		return err
+	}
+
+	if _, err := os.Stat(confDirPath); err != nil {
+		if os.IsNotExist(err) {
+			return nil
+		}
+
+		return err
+	}
+
+	files, err := ioutil.ReadDir(confDirPath)
+	if err != nil {
+		return err
+	}
+
+	defaultRevision, err := getDefaultConfigRevision()
+	if err != nil {
+		return err
+	}
+
+	for _, f := range files {
+		entry := f.Name()
+		if entry == defaultConfigName {
+			continue
+		}
+
+		if entry == defaultRevision {
+			entry = fmt.Sprintf("%s (default)", entry)
+		}
+
+		fmt.Println(entry)
+	}
+
+	return nil
+}
+
+type configSetDefaultCmd struct {
+	Command `name:"set-default" short-description:"Set default config"`
```

Comment:
Maybe `set` would be enough, to make it shorter.
Also, I think it makes more sense to refer to it as the "active" config, not the "default" config. Or "current". But this is personal preference, I don't mind too much one way or the other.